content/en/blog/_posts/2022-02-04-SIG-Multicluster-Spotlight/index.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: blog
+title: "Spotlight on SIG Multicluster"
+date: 2022-02-07
```

Comment:
Ideally, we'd have had the contributor site and main Kubernetes site articles go out on the same publication date - that's what we aim for.

Now the filename and the date _still_ aren't aligned (I might see if someone is willing to make a CI check to point out mismatches).

I'll push a commit to fix that and then merge this.

Reply:
thanks @sftim

Reply:
Thank you, @sftim!